src/components/Layouts/Primary/TopBar/Items/NavMenu/NavMenu.tsx

```diff
@@ -169,254 +153,65 @@ const NavMenu: React.FC = () => {
-                                  : undefined
-                              }
-                              className={clsx(
-                                classes.menuItem,
-                                needsAttention && classes.needsAttention,
-                              )}
-                            >
-                              <Icon
-                                path={tool.icon}
-                                size={1}
-                                className={clsx(
-                                  classes.menuIcon,
-                                  needsAttention
-                                    ? classes.darkText
-                                    : classes.whiteText,
-                                )}
-                              />
-                              <ListItemText
-                                className={clsx(
-                                  needsAttention
-                                    ? classes.darkText
-                                    : classes.whiteText,
-                                )}
-                                primary={t(tool.tool)}
-                              />
-                              {!loading && needsAttention && (
-                                <Box
-                                  className={classes.notificationBox}
-                                  data-testid={`${tool.id}-notifications`}
-                                >
-                                  <Typography>
-                                    {toolData[tool.id].totalCount < 100
-                                      ? toolData[tool.id].totalCount
-                                      : '99+'}
-                                  </Typography>
-                                </Box>
-                              )}
-                            </MenuItem>
-                          );
-                        })}
-                      </Box>
-                    ))}
-                  </MenuList>
-                </ClickAwayListener>
-              </Paper>
-            </Grow>
-          )}
-        </Popper>
-      </Grid>
+      {navPages.slice(0, 3).map((page) => (
+        <Grid key={page.id} item className={classes.navListItem}>
+          <MenuItem
+            component={NextLink}
+            href={page.href ?? ''}
+            tabIndex={0}
+            className={classes.menuItem}
+            aria-current={pathname === page.pathname ? 'page' : undefined}
+          >
+            <ListItemText primary={page.title} />
+          </MenuItem>
+        </Grid>
+      ))}
+
+      <NavMenuDropdown
+        page={navPages[3]}
+        anchorRef={anchorRef}
+        menuOpen={reportsMenuOpen}
+        handleMenuToggle={handleReportsMenuToggle}
+        handleMenuClose={handleReportsMenuClose}
+        testId="ReportMenuToggle"
+        sum={sum}
+        toolData={toolData}
+        loading={loading}
+        isTool={false}
+      />
+      <NavMenuDropdown
```

Comment:
Would it work to define the `anchorRef`, `menuOpen`, and `handleMenuToggle` state inside of the `NavMenuDropdown` component to reduce a little duplication?